content/es/guides/configuration-glossary/configuration-loading-indicator.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'La propiedad loading indicator'
+description: ¡Muestra un indicador de carga chulo mientras se carga la SPA!
```

Comment:
Depending on the Hispanic-speaking country, "Chulo" can have a negative connotation. Please remove it. 

https://dle.rae.es/chulo

Reply:
No caí en esto. Lo sustituí por "atractivo". Hecho!